### PR TITLE
Docs: Beginning of MQTT reference

### PR DESCRIPTION
**Pull request by koenvervloesem**
With the current interest in Rhasspy implementing more of the Hermes protocol, I'm going to document the current MQTT/Hermes implementation of Rhasspy on the reference page. This is just the beginning, I'm going to update it in the next days. You can wait with merging until I have added more.

### CODE REVIEW

**Comment by koenvervloesem** [comment]
Ok, I think this can be merged. It's just to give a better overview of what has been implemented already in Rhasspy: before, this information was scattered around Rhasspy's documentation. Of course this should be kept up-to-date when other topics are implemented.

When more of the Hermes protocol is implemented, I think it's better to split this into its own page to something resembling the [Hermes Reference](https://docs.snips.ai/reference/hermes) that Snips has published. Especially the tables with the JSON payload keys and mosquitto examples are nice and much clearer than the concise descriptions we have now in the Rhasspy reference.

File: docs/reference.md [hunk 1]
```diff
@@ -115,6 +116,34 @@ Application authors may want to use the [rhasspy-client](https://pypi.org/projec
 * `/api/events/log`
     * Listen for log messages published as plain text
 
+## MQTT API
+
+Rhasspy implements part of the [Hermes](https://docs.snips.ai/reference/hermes) protocol. Various services of Rhasspy can be configured to pass along MQTT messages or to react to MQTT messages following the Hermes protocol.
+
+* `hermes/audioServer/<SITE_ID>/playBytes/<REQUEST_ID>`
+    * Rhasspy publishes 16 kHz, 16-bit mono audio in WAV format to this topic.
```
Comment (Romkabouter) [inline docs/reference.md]: This is not entirly correct, if you set TTS to Google Wavenet you can set a different kHz. 
Also, you can select diffent feedback sounds with different audio format, which is played fine by Rhasspy

Comment (koenvervloesem) [reply docs/reference.md]: Thanks for having a look at this! So, the [audio output](https://rhasspy.readthedocs.io/en/latest/audio-output/#mqtthermes) documentation is also not entirely correct then:

> Rhasspy will always try to send 16 kHz, 16-bit mono audio.

Right?

Comment (koenvervloesem) [reply docs/reference.md]: I have changed the wording somewhat now, also in the audio output documentation.

Comment (Romkabouter) [reply docs/reference.md]: I think indeed the audio output is also not completely correct. Good that it's changed

**Pull request merged**